lms/djangoapps/courseware/toggles.py

```diff
@@ -83,6 +83,34 @@
     f'{WAFFLE_FLAG_NAMESPACE}.disable_navigation_sidebar_blocks_caching', __name__
 )
 
+# .. toggle_name: courseware.disable_navigation_sidebar
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Disable navi sidebar on Learning MFE
+# .. toggle_use_cases: temporary
+# .. toggle_creation_date: 2024-03-07
+# .. toggle_target_removal_date: None
+# .. toggle_tickets: AXIMST-611
+# .. toggle_warning: None.
+COURSEWARE_MICROFRONTEND_SIDEBAR_DISABLED = CourseWaffleFlag(
+    f'{WAFFLE_FLAG_NAMESPACE}.disable_navigation_sidebar', __name__
+)
+
+# .. toggle_name: courseware.show_default_right_sidebar
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: If waffle flag disabled
+# Discussions or Notifications sidebar shouldn't be displayed at all on Learning MFE.
+# If waffle flag enabled - Discussions opens always on the pages with discussions,
+# if user is in Audit and course has verified mode - show Notifications.# .. toggle_use_cases: temporary
+# .. toggle_creation_date: 2024-04-28
+# .. toggle_target_removal_date: None
+# .. toggle_tickets: FC-0056
+# .. toggle_warning: None.
+COURSEWARE_SHOW_DEFAULT_RIGHT_SIDEBAR = CourseWaffleFlag(
```

Comment:
```suggestion
COURSEWARE_MICROFRONTEND_ALWAYS_OPEN_AUXILIARY_SIDEBAR = CourseWaffleFlag(
```

See other comment below.